test/extended/util/proxy.go

```diff
@@ -68,7 +68,9 @@ func RunProxy() {
 	addr := flag.String("127.0.0.1", ":8888", "proxy listen address") // using network-mode=user
 	flag.Parse()
 	proxy.Verbose = *verbose
-	proxy.Logger = log.StandardLogger()
+	l := log.New()
+	l.Out = f
+	proxy.Logger = l
```

Comment:
This seems out of place in `integration: cleanup to AfterAll and BeforeSuite`.
Is this required to fix something?

Reply:
Fair, it is out of place. I was reading something about logging and felt the need to change this asap :). Will remove it from this PR! Thanks